.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: objective-c
+osx_image: xcode8.3
```

Comment:
Travis CI uses OS X 10.11.6 and Xcode 7.3.1 by default but project using Xcode8.3